metashare/api/models.py

```diff
@@ -90,3 +94,24 @@ def valid_token_for(self):
         if all(self.token) and self.org_id:
             return self.org_id
         return None
+
+
+class Product(HashIdMixin):
+    name = models.CharField(max_length=50, unique=True)
+    repo_name = models.SlugField(unique=True)
+    version_number = models.CharField(max_length=50)
```

Comment:
I guess this is here because it's in the mockup, but I don't think this is very relevant for MetaShare, which is focused on development, not releasing versions. Showing a last modified date would probably be more useful.

Reply:
@wlonk You'll go ahead and remove/replace this?

Reply:
I removed `version` and added a task for @wlonk to add last-modified date.